src/stories/css/radio-button.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+    font-family: 'OpenSans';
+    src: url('../assets/Fonts/Open_Sans/OpenSans-Regular.ttf')  format('truetype');
+}
+
+.radio--label {
+    font-family: 'OpenSans', sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    margin-left: 4px;
+    color: #3F3F3F;
+}
+
```

Comment:
Add a CSS class for the hover functionality. Right now the component does not change when hovered over.